internal/services/applications/migrations/application_resource.go

```diff
@@ -374,6 +375,7 @@ func ResourceApplicationInstanceResourceV0() *schema.Resource {
 													"emit_as_roles",
 													"include_externally_authenticated_upn",
 													"include_externally_authenticated_upn_without_hash",
+													"max_size_limit",
```

Comment:
On a second pass, we don't need to add this here as this should be a representative snapshot of the schema available at that time.
```suggestion
```